test/titan_run_tests.pbs

```diff
@@ -17,16 +19,23 @@
 # Required since some GPU tests run multiple MPI tasks per node.
 export CRAY_CUDA_MPS=1
 
-# Change to the directory from which the batch job was submitted, i.e the top-level source
-# directory.
-cd $PBS_O_WORKDIR
+# Check where the script has been submitted from and change to the top-level source dir (DCA).
+if [ "${PBS_O_WORKDIR: -4}" = "test" ]; then
+    cd $PBS_O_WORKDIR/..
+elif [ "${PBS_O_WORKDIR: -3}" == "DCA" ]; then
+    cd $PBS_O_WORKDIR
+else
+    1>&2 printf "Usage: qsub titan_run_tests.lsf      [from test dir]\n       qsub test/titan_run_tests.lsf [from DCA dir]\n"
```

Comment:
Aren't all the tests: *.pbs ? are we switching the scripts to lsf ?

Reply:
Copy & paste leftover.
Fixed in be48e5aa.